data/resources/data/TileData.json

```diff
@@ -14,7 +14,8 @@
         },
         "title": "Coal Pile",
         "upkeep cost": 0,
-        "water": 0
+        "water": 0,
+        "isOverPlacable": 0
```

Comment:
Is there a reason this is an int and not a bool? I'm not against it, I'm mostly curious

Reply:
no reason actually, will fix that and use boolean in next PR